components/automate-ui/src/app/entities/clients/client.effects.ts

```diff
@@ -129,4 +133,33 @@ export class ClientEffects {
       });
     }));
 
+  @Effect()
+  resetKeyClient$ = this.actions$.pipe(
+    ofType(ClientActionTypes.RESETKEY),
+    mergeMap(( { payload: { server_id, org_id, name } }: ResetKeyClient) =>
+      this.requests.resetKeyClient(server_id, org_id, name).pipe(
+        map((resp: ResetKeySuccessPayload) => new ResetKeyClientSuccess(resp)),
+        catchError((error: HttpErrorResponse) =>
+          observableOf(new ResetKeyClientFailure(error))))));
+
+  @Effect()
+  resetKeyClientSuccess$ = this.actions$.pipe(
+    ofType(ClientActionTypes.RESETKEY_SUCCESS),
+    map(({ payload: { name } }: ResetKeyClientSuccess) => {
+      return new CreateNotification({
+        type: Type.info,
+        message: `Successfully resetting Client key - ${name}.`
```

Comment:
I see there are a couple instances like you have here, but let's move to be consistent with the majority:
```suggestion
        message: `Reset client key ${name}.`
```
1. Deleting "Successfully".
2. Remove hyphen.

Would be helpful if you could fix the other five occurrences of that pattern in the code base.

Reply:
This format is suggested by UX team, will go with "Successfully resetting Client key - ${name}."